### ISSUE TITLE
R Window Bug

### ISSUE THREAD

**Issue by Patowhiz** [IDEMSInternational/R-Instat#4124]
@africanmathsinitiative/developers  When i display a graph through the **R window** and **scroll** through, an error that breaks the application occurs(System.Runtime.InteropServices.SEHException). 
Can someone check if this happens in their machines too. It seems like on scroll event there is an issue in System.Windows.Forms.dll 

![error3](https://user-images.githubusercontent.com/21179007/32377293-322f9f9c-c0b8-11e7-8110-4fdd636c87d2.JPG)

**Comment by dannyparsons** [comment]
Yes this is a very annoying bug with RDotNet which we would require changing the RDotNet package itself to fix. It does seem to be in active development again so we could actually write an issue on their GitHub and see if they can fix it.
